adapters/mobfoxpb/mobfoxpbtest/params/banner.json

```diff
@@ -0,0 +1,3 @@
+{
+  "TagID": "6"
```

Comment:
Only the `params/race/*.json` files are needed. You can safely remove this one. I noticed that a few other adapters do this. I will work on getting them removed but can you please do it for yours? Thanks!